src/input-MicroMoB.ts

```diff
@@ -78,3 +78,12 @@ export interface MicroMoBBHRMInput {
    */
   adult_Z: number[]
 }
+
+
+/**
+ * @title Batch input for running MicroMoB with Beverton-Holt aquatic mosquito model and 
+ * Ross-Macdonald adult mosquito model over multiple sets of parameters.
+ */
+export interface MicroMoBBHRMInputBatch {
```

Comment:
When I implemented Simon's suggestion for the SIR models, I actually modified this to be just `export type MinimalModelInputBatch = MinimalModelInput | MinimalModelInput[]`, i.e. without the wrapper object. I think I had a justification for this at the time that then turned out to be moot, but is it worth doing the same thing here for consistency?

A similar thing applies for the `MicroMoBBHRMOutputBatch`.

Reply:
Thanks for the review @steven-cd, I just pushed those changes.